tests/integration/test_uninstall.py

```diff
@@ -127,7 +130,6 @@ def test_uninstall_all_dev(pipenv_instance_private_pypi):
 
         c = p.pipenv("uninstall --all-dev")
         assert c.returncode == 0
-        assert p.pipfile["dev-packages"] == {}
```

Comment:
I think I need to revert this part of the PR -- while it doesn't make sense to me in terms of the code paths that --all-dev does something different than --all, I guess it does.